stable/artifactory-ha/README.md

```diff
@@ -443,6 +468,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.customInitContainers`| Custom init containers                  |                                            |
 | `artifactory.customSidecarContainers`| Custom sidecar containers            |                                            |
 | `artifactory.customVolumes`       | Custom volumes                    |                                                  |
+| `artifactory.userPluginSecrets`   | Secret name array for Artifactory user plugins |                                     |
```

Comment:
Should be "Array of secret names for Artifactory user plugins"

Reply:
@eldada done